src/JDS.jl

```diff
@@ -69,6 +69,7 @@ using Reexport: @reexport
 end # @reexport
 
 const SMALL_IM_ATTR = "width=70%"
+const download = Downloads.download
```

Comment:
This is hidden to the reader. Maybe we should just keep it as it used to be? The suggestion to use `using Foo: bar` is a suggestion and not meant to be a strict rule.

Reply:
This is just the intent, this is hidden. I am trying to make the:

```julia
using Downloads: download

url = "https://raw.githubusercontent.com/JuliaDataScience/JuliaDataScience/main/Project.toml"
my_file = download(url) # tempfile() being created
```

in `julia_basics.md` to work.

Reply:
That will not work for the reader either. In Julia 1.6, there are `Base.download` and `Download.download`:
```
help?> Base.download
  download(url::AbstractString, [path::AbstractString = tempname()]) -> path

  Download a file from the given url, saving it to the location path, or if not specified, a temporary path. Returns the path of the downloaded file.

  │ Note
  │
  │  Since Julia 1.6, this function is deprecated and is just a thin wrapper around Downloads.download. In new code, you should use that function directly instead
  │  of calling this.
```

Reply:
```suggestion
```